Recruitment/JobSpecs/why_us.md

```diff
@@ -1,34 +1,17 @@
 
-We partner with clients to build new products and services, and help their teams scale independently.
-
-We are not a consultancy, and not an outsourcer. We are a hands-on partner that participates and cares about the detail. The Buildit team invests significantly with every client we work with, we focus on getting things done, helping prove a method and a model to unlock future speed and scale.
-
-We specialise in customer centricity, high calibre engineering and flow.
+We are not a consultancy, and not an outsourcer. We are a hands-on partner that participates and cares about the detail. We recognise that real change comes from people doing things differently, rather than people doing the same things differently. The Buildit team invests significantly with every client we work with, focusing on getting things done and helping prove a method and a model to unlock future speed and scale. We specialise in customer centricity, high calibre engineering and flow.
 
 We have done this before, transforming large organisations with millions of users. And now we’re applying what we’ve learned to help create the next generation of business.
 
-###Why us?
-Do you think there’s something wrong with the way most companies approach problems, whether they’re people, process, or technology related?
-
-Do you want to help them figure out how to think differently?
-
-Do you love technology, but understand that real change comes from people doing things differently, rather than people doing the same things differently?
-
-We’re looking for people who want to help us change the way companies think and approach problems; people who want to work with teams and individuals who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or poor customer experience.
-
-We’re looking for people who want to be treated like adults – who neither need, nor want, someone else to tell them what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it. We’re looking for people who are comfortable finding their own place.
-
-We’re looking for people who are willing to seek input when they want to make a decision that will affect others, or when others know more than they do.
 
-We build software. We build teams. We build new ways of working. We build better organisations.
+### Our approach
+We are looking for people who want to help us change the way companies think and approach problems; people who want to work as individuals and cross-functional teams, who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or poor customer experience.

```

Comment:
This sentence is still very complicated and you lose the trend of thought part way through. We need to simplify the emotions here IMO.